src/main/java/uk/gov/hmcts/reform/logging/MdcFields.java

```diff
@@ -2,6 +2,7 @@
 
 import org.slf4j.MDC;
 
+@SuppressWarnings("HideUtilityClassConstructor")
```

Comment:
IMHO we should either remove this rule altogether or add private constructors to those classes

Reply:
and I'm leaning towards adding those private constructors :)